clicommand/agent_start.go

```diff
@@ -1034,7 +1038,7 @@ func handlePoolSignals(ctx context.Context, l logger.Logger, pool *agent.AgentPo
 
 		for sig := range signals {
 			l.Debug("Received signal `%v`", sig)
-			setStatus(fmt.Sprintf("Recieved signal `%v`", sig))
+			setStatus(fmt.Sprintf("Received signal `%v`", sig))
```

Comment:
😅

Reply:
So rare for me to be on the other side of this 😂